api_backend.yaml

```
@@ -476,6 +476,28 @@ paths:
           description: Unavailable service
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/token":
```

```suggestion
  "/token/support":
```
or
```suggestion
  "/support_token":
```
Perhaps we would add other tokens in the future.
Could it be a better name?

---

Initially it was `"/support-token` https://github.com/pagopa/io-backend/pull/726#discussion_r522191997, but we have changed it in order to mantain a generic approach, but you're right cc: @BurnedMarshal @gunzip